plasmapy/formulary/collisions.py

```diff
@@ -111,130 +109,261 @@ def Coulomb_logarithm(
+
+    Options 1–4 are straight-line Landau-Spitzer (``"ls..."``) methods in which the trajectory of a
+    Coulomb collision is modeled as a straight line. For the straight-line Landau-Spitzer methods, the Coulomb
+    logarithm (:math:`\ln{\Lambda}`) is defined to be:
 
     .. math::
         \ln{\Lambda} \equiv \ln\left( \frac{b_{max}}{b_{min}} \right)
 
-    where :math:`b_{min}` and :math:`b_{max}` are the inner and outer
-    impact parameters for Coulomb collisions [1]_.
+    Options 5–7 are hyperbolic Landau-Spitzer (``"hls..."``) methods in which the trajectory of a
+    Coulomb collision is modeled as a hyperbola. For the hyperbolic Landau-Spitzer methods, the Coulomb
+    logarithm (:math:`\ln{\Lambda}`) is defined to be:
 
-    The outer impact parameter is given by the Debye length:
-    :math:`b_{min} = \lambda_D` which is a function of electron
-    temperature and electron density.  At distances greater than the
-    Debye length, electric fields from other particles will be
-    screened out due to electrons rearranging themselves.
+    .. math::
+        \ln{\Lambda} \equiv \frac{1}{2} \ln\left(1 + \frac{b_{max}^2}{b_{min}^2} \right)
 
-    The choice of inner impact parameter is either the distance of closest
-    approach for a 90 degree Coulomb collision or the thermal deBroglie
-    wavelength, whichever is larger. This is because Coulomb-style collisions
-    cannot occur for impact parameters shorter than the deBroglie
-    wavelength because quantum effects will change the fundamental
-    nature of the collision [2]_, [3]_.
+    For all 7 methods, :math:`b_{min}` and :math:`b_{max}` are the inner impact parameter and the outer
+    impact parameter for Coulomb collisions [1]_; :math:`b_{min}` and :math:`b_{max}` are each computed
+    by `impact_parameter`, another function.
 
-    Errors associated with the classical Coulomb logarithm are of order its
-    inverse. If the Coulomb logarithm is of order unity, then the
-    assumptions made in the standard analysis of Coulomb collisions
-    are invalid.
+    .. note::
+        For strongly-coupled plasma, PlasmaPy recommends Option 7, ``"hls_full_interp"`` or ``"GMS-6"``,
+        because of its high accuracy regardless of a plasma's strength of coupling.
 
-    For dense plasmas where the classical Coulomb logarithm breaks
-    down there are various extended methods. These can be found
-    in D.O. Gericke et al's paper, which has a table summarizing
-    the methods [4]_. The GMS-1 through GMS-6 methods correspond
-    to the methods found it that table.
+    **Explanation of PlasmaPy-Supported Methods of Computing the Coulomb Logarithm**
 
-    It should be noted that GMS-4 thru GMS-6 modify the Coulomb
-    logarithm to the form:
+    In this section, further information about each method, such as about interpolation and other special
+    features, is documented. Please refer to Reference [4]_ for additional information about these methods.
```

Comment:
```suggestion
    In this section, further information about each method, such as about
    interpolation and other special features, is documented. Please refer
    to Reference [4]_ for additional information about these methods.
```
Inconsequential formatting suggestion for line length